frontend/src/pages/instance/Snapshots/dialogs/SnapshotAssignDialog.vue

```diff
@@ -104,16 +105,16 @@ export default {
             this.submitted = false
         },
         fetchData: async function () {
-            const data = await snapshotApi.getInstanceSnapshots(this.project.id)
-            this.snapshots = data.snapshots
+            const data = await snapshotApi.getInstanceSnapshots(this.instance.id)
+            this.snapshots = (data.snapshots || []).sort((a, b) => a.name.localeCompare(b.name))
```

Comment:
Snapshots should be ordered by date of creation - as they are a history, rather than name.

If the order is inconsistent, that should be resolved in the backend rather than the api.